app/src/main/java/org/hackillinois/android/database/entity/Project.kt

```diff
@@ -18,11 +21,70 @@ data class Project(
 ) {
     fun getMentorsString(): String {
         var mentors = ""
-        for (i in 0..this.mentors.lastIndex - 1) {
+        for (i in 0 until this.mentors.lastIndex) {
             mentors += this.mentors[i] + ", "
         }
         mentors += this.mentors[this.mentors.lastIndex]
 
         return mentors
     }
+
+    @Ignore
+    private val mapResourceMappings = mapOf(
+        "Siebel_0" to R.drawable.siebel_floor_0,
+        "Siebel_1" to R.drawable.siebel_floor_1,
+        "Siebel_2" to R.drawable.siebel_floor_2,
+        "ECEB_1" to R.drawable.eceb_floor_1,
+        "ECEB_2" to R.drawable.eceb_floor_2,
+        "ECEB_3" to R.drawable.eceb_floor_3,
+        "DCL" to R.drawable.dcl,
+        "Kenney" to R.drawable.kenney
+    )
+
+    fun getBuildingName(): String {
+        val split = room.split(" ")
+        return if (split.isEmpty()) {
+            ""
+        } else {
+            split[0]
+        }
+    }
+
+    @Ignore
+    private val NO_RESOURCE = 0
+
+    fun getIndoorMapResource(): Int {
+        val split = room.split(" ")
+        if (split.isEmpty()) {
+            return NO_RESOURCE
+        }
+
+        var buildingKey = split[0]
+        if (split.size >= 2) {
```

Comment:
Siebel Atrium is also a valid location... I guess if it starts with "Siebel", look for a 0, then look for a 2, else default to 1st floor. Same thing with ECEB Atrium.

Reply:
Will there ever be projects located in Kenney or DCL? Should I just remove those?

Reply:
I removed them, lmk if it should be different